harness/determined/common/experimental/workspace.py

```diff
@@ -79,6 +79,34 @@ def get_with_offset(offset: int) -> bindings.v1ListRPsBoundToWorkspaceResponse:
 
         return [resource_pool.ResourcePool(self._session, name=rp) for rp in resource_pool_names]
 
+    def create_project(self, name: str) -> project.Project:
+        """Creates a new project in this workspace with the provided name.
+
+        Args:
+            name: The name of the project to create.
+
+        Returns:
+            The newly-created :class:`~determined.experimental.Workspace`.
+
+        Raises:
+            errors.APIException: If the project with the passed name already exists.
+        """
+        req = bindings.v1PostProjectRequest(name=name, workspaceId=self.id)
```

Comment:
wonder if some of the other parameters `v1PostWorkspaceRequest` takes could be useful at some point. like setting the default resource pools or user groups. 

just a thought, i think this covers 95% of use cases now.

Reply:
Set description because that's all the REST API can currently do on project creation.